doc/faq.md

```diff
@@ -153,31 +111,19 @@ address of 10.61.0.25, and VLAN102 being used for client traffic.
           parameters:
             stp: false
 
-#### Things to note
+Note the following details:
```

Comment:
This whole section feels like it should be removed/moved to a tutorial in the forums/be rewritten to cover a specific scenario, as its apparently geared toward l2 bridging, but without a guide on how to do that, it has no real context. 

Also many users stick with the private lxdbr0 bridge, so its not relevant in the most common case.

Reply:
Sounds good